package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Use this to save yourself some time when working on a webpack configuration.",
   "main": "index.js",
   "scripts": {
-    "cover": "nyc --reporter=lcov --reporter=text npm t",
+    "cover": "nyc --reporter=lcov --reporter=text ava --coverage",
```

Comment:
Could you please check your `node` and `npm` versions and report them back here? Like

```
$ node --version
v5.8.0
$ npm --version
3.8.1
```

There seems to be an issue with `nyc` running on node `5.8.0` and npm `3.7.3`, as reported [here](https://github.com/bcoe/nyc/issues/190).